frb_example/pure_dart/rust/src/api.rs

```diff
@@ -779,3 +781,10 @@ pub fn multiply_by_ten(measure: Measure) -> Option<Measure> {
         },
     }
 }
+
+pub fn call_both_module_system() -> String {
+    let n = use_new_module_system(1);
```

Comment:
Seems that we are not using the structs in the old/new module systems? If I understand correctly, this PR tries to allow frb to *find* the structs.

Thus, what about something like `call_both_module_system(a: StructInNewModuleSystem, b: StructInOldModuleSystem)`

Reply:
Fix that. But I found there's one restriction: `sub_module` must be `pub` currently, because generated code does not consider `pub use`. But that's another issue. ^_^